ocpkg

```diff
@@ -902,6 +904,17 @@ rm -rf master.tar.gz ros-behavior-scripting-master/
 cd $CURRENT_DIR
 }
 
+# atomspace unit tests have UTF-8 test cases
+ensure_UTF_installed(){
+    if locale -a | grep -q en_US.utf8
+    then
```

Comment:
it doesn't have to be en_US - it can be any utf8 local.  So it should be `grep -q utf8` instead.

Reply:
True, it can be any utf8, but unit test ( per my change) loads en_US, the unit test could be smarter to match. 
/ed